src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/ClaimantResponseContentProvider.java

```diff
@@ -102,7 +102,7 @@ private void addFormalisedOption(
         Map<String, Object> content,
         ResponseAcceptation responseAcceptation
     ) {
-        switch (responseAcceptation.getFormaliseOption()) {
+        switch (responseAcceptation.getFormaliseOption().get()) {
```

Comment:
`Optional.get()` without checking its presence. Replace your `Optional.get()` calls with `orElse` or `orElseThrow` baed on the scenario.